.github/workflows/release.yaml

```diff
@@ -232,15 +229,14 @@ jobs:
           name: wheels
```

Comment:
if you expand above this line, we pass no argument to release here, target is specified on its own line.